config/rewrites.d/docs.rackspace.com.json

```diff
@@ -0,0 +1,67 @@
+{
+	"docs.rackspace.com": [
+		{
+			"description": "Redirect root traffic to developer.rs.com/docs/",
+			"from": "^\\/$",
+			"to": "/docs/",
+			"toProtocol": "https",
+			"toHostname": "developer.rackspace.com",
+			"rewrite": false,
+			"status": 302
+		},
+		{
+			"description": "cloud servers legacy",
+			"from": "^\\/servers/api/v1.0/cs-devguide\\/$",
```

Comment:
@kenperkins This "from" regex only catches this URL if it includes a trailing slash. Perhaps change the end of this regex to `\\/?$`?